lib/layout/dummyLayoutV2.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * dummylayoutV2.js
+ *
+ * this creates a basic poorly laid out graph
+ *
+ * @author John O'Brien.
+ */
+
+var Node = require('../model/node.js');
+var Graph = require('../model/graph.js');
+var Port = require('../model/port.js');
+
+// Create a graph instance.
+
+function buildGraph(graphs) {
+  var nodes = graph.nodes;
+  //loop for building the nodes
+  for(var i = 0; i < nodes.length; i++){
+    nodes[i].x = i * 2;
+    nodes[i].y = i * 2;
```

Comment:
Use setPos rather than setting the properties directly. Also, those properties don't exist on Node.